vic/drivers/shared_all/src/initialize_lake.c

```diff
@@ -149,14 +63,34 @@ initialize_lake(lake_var_struct  *lake,
         lake->runoff_out = 0.0;
         lake->snowmlt = 0.0;
         lake->vapor_flux = 0.0;
-    } // if (!skip_hydro)
 
-    // Initialize other miscellaneous lake properties
-    lake->aero_resist = 0;
-    for (k = 0; k < lake->activenod; k++) {
-        lake->density[k] = CONST_RHOFW;
+        // Initialize other terms
+        lake->aero_resist = 0;
+        lake->soil.pot_evap = 0.0;
     }
 
+    // Initialize other lake state variables
+    lake->areai = 0.0;
+    lake->coldcontent = 0.0;
+    for (i = 0; i < MAX_LAKE_NODES; i++) {
+        lake->density[i] = CONST_RHOFW;
+    }
+    lake->hice = 0.0;
+    lake->ice_throughfall = 0.0;
+    lake->new_ice_area = lake->areai;
+    lake->pack_temp = 0.0;
+    lake->pack_water = 0.0;
+    lake->SAlbedo = 0.0;
+    lake->sdepth = 0.0;
+    lake->surf_temp = 0.0;
+    lake->surf_water = 0.0;
+    lake->swe = 0.0;
+    for (i = 0; i < MAX_LAKE_NODES; i++) {
+        lake->temp[i] = 0;
+    }
```

Comment:
consider also setting `lake->temp` in the for loop at line 75 when setting `density`.